example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -483,7 +483,11 @@ describe('The S3 Ingest Granules workflow', () => {
     });
 
     it('triggers the granule record being added to DynamoDB', async () => {
-      const record = await granuleModel.get({ granuleId: inputPayload.granules[0].granuleId });
+      const record = await waitForModelStatus(
```

Comment:
I just noticed that we also have a `waitUntilGranuleStatusIs` helper that is used elsewhere in this file. I guess it doesn't matter, though maybe it would be nice to have one helper for handling this sort of thing?